pkg/api/validate/openshiftcluster_validatedynamic.go

```diff
@@ -86,6 +88,12 @@ func (dv *openShiftClusterDynamicValidator) Dynamic(ctx context.Context, oc *api
 		return err
 	}
 
+	vnetManager := utilvnet.NewManager(r.SubscriptionID, fpAuthorizer)
```

Comment:
instead of utilvnet, do `spVnet := network.NewVirtualNetworksClient(r.SubscriptionID, spAuthorizer)` above with the other `sp*` variables